zips/zip-0321.rst

```diff
@@ -212,6 +269,10 @@ For example,
 The amount MUST NOT be greater than 21000000 ZEC (in general, monetary amounts
```

Comment:
If this rule is enforced at the parser level at present, it means that parsers that add support for `req-assetId` must be cautious to also relax this parsing restriction.

Reply:
In ZIP Editing, we identified that an additional `req-assetamount` parameter should be added. This should be base64 encoded, and should specify the value in the minimal unit. A URI that has both `amount` and `req-assetamount` should be rejected.

For assets other than ZEC, we have no way of determining how many decimal places there are.

An alternative suggested by @daira is that instead of two fields, a base64-encoded `req-asset` that encodes both the asset ID and the value should be used, because in the context of an opaque asset ID, the user has no frame of reference for interpreting the value of a `req-assetamount` field.